interface/controller/login_controller.go

```diff
@@ -41,7 +41,7 @@ func (c *loginController) Login() (*model.Result, error) {
 
 	result, err := loginResultUsecase.LoginResult(c.studentID, c.password)
 	if err != nil {
-		logs.Error.Panicln(err)
+		logs.Warning.Println(err)
```

Comment:
I think there should be more detailed error handling here. For example, if the returned status code was `500`, this would be a `logs.Error` situation. If it's a `404` (user not found), it would be `logs.Warning`, etc.